test/utils.jl

```diff
@@ -160,20 +160,11 @@ function copy_test_package(tmpdir::String, name::String; use_pkg=true)
         end
     end
 end
-function add_test_package(name::String, uuid::UUID)
-    test_pkg_dir = joinpath(@__DIR__, "test_packages", name)
-    spec = Pkg.Types.PackageSpec(
-        name=name,
-        uuid=uuid,
-        path=test_pkg_dir,
-    )
-    Pkg.add(spec)
-end
 
 function add_this_pkg()
     pkg_dir = dirname(@__DIR__)
     pkg_uuid = Pkg.TOML.parsefile(joinpath(pkg_dir, "Project.toml"))["uuid"]
-    spec = Pkg.Types.PackageSpec(
+    spec = Pkg.PackageSpec(
```

Comment:
@staticfloat some of these tests were set up incorrectly. I made a comment earlier but perhaps it got drowned out by everything else (the earlier comment is here: https://github.com/JuliaLang/Pkg.jl/pull/1372#discussion_r323056817).

Reply:
Yes, it did get drowned out.  Do you still see the overriding test error with this change?  That error message means that Julia did not load the `Overrides.toml` file that it should have, which probably means that the depot path wasn't set correctly, or something like that, since it keys off of depot paths in order to determine which override file it should be paying attention to.

Reply:
I am not able to reproduce it anymore. :man_shrugging: I guess I'll just keep an eye out for it

Reply:
You can see a stack trace here: https://travis-ci.org/JuliaLang/Pkg.jl/jobs/588723217.
All I did was change `add` to `develop` and used the API facing `PackageSpec`, since that seemed like the most straightforward mapping.

I'm reasonably sure I also tried `add` (tracking by repo) and I hit the same error. Naively, this makes it seem unlikely that a bad `DEPOT_PATH` is the issue.